src/util/strencodings.h

```diff
@@ -29,6 +29,22 @@ enum SafeChars
     SAFE_CHARS_URI, //!< Chars allowed in URIs (RFC 3986)
 };
 
+/**
+ * Used by ParseByteUnits()
+ * SI are 1000 base while IEC is 1024 base
+ */
+enum ByteUnitMultiplier {
+    NOOP = 1ULL,
+    K_SI = 1000ULL,
+    K_IEC = 1024ULL,
+    M_SI = 1'000'000ULL,
+    M_IEC = 1ULL << 20,
+    G_SI = 1'000'000'000ULL,
+    G_IEC = 1ULL << 30,
+    T_SI = 1'000'000'000'000ULL,
+    T_IEC = 1ULL << 40,
+};
```

Comment:
I use the "technical" terms but I don't think they are intuitive, open to suggestions.